app/views/layouts/application.html.erb

```diff
@@ -13,8 +13,11 @@
 
     <section class="top-bar-section">
       <ul class="right">
-        <%- if current_user -%>
+        <%- if user_signed_in? -%>
           <li><%= link_to 'Sign Out', destroy_user_session_path, method: :delete %></li>
+          <% if current_user.admin? %>
+            <li><%= link_to "Admin Section", root_path %></li>
+          <% end %>
         <%- else -%>
           <li><%= link_to 'Sign Up', new_user_registration_path %></li>
           <li><%= link_to 'Sign In', new_user_session_path %></li>
```

Comment:
I would like to include a feature test making sure an admin and default member are seeing the correct links